src/tests/ftest/nvme/nvme_pool_extend.yaml

```diff
@@ -0,0 +1,75 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+  test_clients:
+    - client-E
+    - client-F
+    - client-G
+# If we define the server under test_servers, launch.py will convert it to the
+# actual server name passed into --test_servers. If above test_servers is hosts,
+# it'll be used as one of the servers at test startup time, so use something
+# other than hosts.
+extra_servers:
+  test_servers:
+    - server-D
+timeout: 1000
+server_config:
+    servers_per_host: 2
+    name: daos_server
+    servers:
+      0:
+        pinned_numa_node: 0
+        nr_xs_helpers: 1
+        fabric_iface: ib0
+        fabric_iface_port: 31416
+        log_file: daos_server0.log
+        bdev_class: nvme
+        bdev_list: ["0000:81:00.0"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+        scm_mount: /mnt/daos0
+      1:
+        pinned_numa_node: 1
+        nr_xs_helpers: 1
+        fabric_iface: ib1
+        fabric_iface_port: 31417
```

Comment:
Can you change this I just got email from Alex that we should not use +1 port in case of Two sever.